ansible/roles/products/hand-e/server/desktop-icons/defaults/main.yml

```diff
@@ -34,3 +34,11 @@ arm_hand_readthedocs_link: ""
 scene_file: ""
 arm_mass_payload: 5.0
 hand_side_ad: ""
+bimanual_warning_title: "Unimanual Launch Warning"
+bimanual_warning_text: "It is potentially unsafe to run a bimanual robot in unimanual mode; the pose of the unused arm is unknown, which may result in collisions.\nTo safely launch in unimanual mode, please first launch in bimanual mode, and use Rviz MotionPlanning to move the arm and hand you do not intend to use into the 'safe_store' state.\nBy clicking 'I accept the risks', you acknowledge that you understand the risks involved and take full responsibility for any consequences that may arise."
```

Comment:
```suggestion
bimanual_warning_text: "It is potentially unsafe to run a bimanual robot in unimanual mode; the pose of the unused arm is unknown, which may result in collisions.\nTo safely launch in unimanual mode, please first launch in bimanual mode, and use Rviz MotionPlanning to move the arm and hand you do not intend to use into the 'safe_store' state.\nBy clicking 'I accept the risks', you acknowledge that you understand the risks involved by running the unimanual system without moving the other robot arm and take full responsibility for any consequences that may arise."
```

Reply:
Otherwise it sounds like even if they did do what we suggested they still accept some risk

Reply:
By the way, how sure are we that they can't collide with the other arm even in "safe store mode"?

Reply:
I believe the scene was tested by omega

Reply:
@georgiablanco

Reply:
scenes were tested and approved in another PR